Detectors/EMCAL/reconstruction/src/CaloRawFitter.cxx

```diff
@@ -193,9 +223,9 @@ std::tuple<short, short, short> CaloRawFitter::selectBunch(const std::vector<Bun
   }
 
   if (bunchindex >= 0) {
-    bool bunchOK = checkBunchEdgesForMax(bunchvector.at(bunchindex));
+    bool bunchOK = checkBunchEdgesForMax(bunchvector[bunchindex]);
     if (!bunchOK) {
-      bunchindex = -1;
+      throw RawFitterError_t::BUNCH_NOT_OK;
```

Comment:
Possible source for crashes in the full CI test

Reply:
This is the source for the full CI crash - we will investigate this further but for the moment the exception is caught in the RawToCellConverter.